### PR TITLE
404 fix

### PR DESCRIPTION
**Pull request by traugertyson**
Attempting to fix issue #56 . Pretty much copy pasted this from the old liquid 404.

### CODE REVIEW

File: app/views/desktop/404.ejs [hunk 1]
```diff
@@ -2,7 +2,97 @@
...
+            "Now, please let me sulk alone.",
+            "I'm so depressed."
+            );
+            var speed=60;
+            var index=0; text_pos=0;
+            var str_length=tl[0].length;
+            var contents, row;
+
+    function type_text()
+    {
```
Comment (sameetandpotatoes) [inline app/views/desktop/404.ejs]: Minor, but can you make sure `{` are on same lines as function declarations, if statements, etc?

And put braces around `while` and `if` statements that are just one line long as well?

**Comment by traugertyson** [comment]
Ok, I think I changed them all. I take it that is the style that should be used?

**Comment by sameetandpotatoes** [comment]
Thanks @bcongdon for pointing that out. This looks good now 👍

**Comment by bcongdon** [comment]
Thanks! 🎉

**Pull request merged**